src/platform/silabs/SiWx917/CHIPCryptoPALTinyCrypt.cpp

```diff
@@ -106,7 +106,9 @@ typedef struct
     uint8_t public_key[2 * NUM_ECC_BYTES];
 } mbedtls_uecc_keypair;
 
+#if !(SLI_SI91X_MCU_INTERFACE)
```

Comment:
This is the SiWX specific file - when would this define not be set?

Reply:
I'm not sure i understand the comment - the define is the 917 SoC and that platform is the only to use that file if i am not mistaken?

Reply:
okay, so if a platform doesn't support TRNG and wanted to use the tinycrypt then this can be borrowed.

Reply:
Removed the unused code, I thought maybe in the future we can use it when using tinycrypt but yeah not worth to keep this since this will never be executed.